src/cli/commands/config.ts

```diff
@@ -0,0 +1,66 @@
+import * as snyk from '../../lib';
+
+const validMethods = ['set', 'get', 'unset', 'clear'];
+
+export = async function config(
```

Comment:
Didn't you just say this:

> General small comment about import / export. Please prefer named exports i.e. export { foo } or export function foo, instead of default (export default foo) or namespace exports (export = foo).

Don't we want to follow it a different export to avoid `import x = require('x')`?

```suggestion
export async function config(
```

Reply:
I did! The problem here is that config is hot-required, so it needs to use older mechanisms, as far as I can tell. I think this may actually need to end up as a default export.